### ISSUE TITLE
Error execution failed

### ISSUE THREAD

**Issue by tb582** [LukeMurphey/splunk-website-monitoring#37]
Seeing these in the logs as of splunk 8.0.1 and latest monitoring upgrade:

Traceback (most recent call last):
--
  | 1/14/2012:08:57.053 PM | 2020-01-14 17:08:57,053 ERROR Execution failed
  | 1/14/2012:08:51.160 PM | Traceback (most recent call last):
  | 1/14/2012:08:51.160 PM | 2020-01-14 17:08:51,160 ERROR Execution failed
  | 1/14/2012:08:49.215 PM | Traceback (most recent call last):
  | 1/14/2012:08:49.215 PM | 2020-01-14 17:08:49,215 ERROR Execution failed
  | 1/14/2012:08:47.160 PM | Traceback (most recent call last):
  | 1/14/2012:08:47.160 PM | 2020-01-14 17:08:47,160 ERROR Execution failed
  | 1/14/2012:08:45.247 PM | Traceback (most recent call last):

**Comment by LukeMurphey** [comment]
Can you provide the full traceback?

Also, let me know what version of the app you are running.

**Comment by tb582** [comment]
Where do I get the full traceback? - running 2.8.0

**Comment by LukeMurphey** [comment]
Most likely, the traceback was broken into several lines. Try searching for other logs right around the time of the event containing the word "traceback".

**Comment by tb582** [comment]
```

1/14/2012:18:17.749 PM | NameError: global name 'splunk' is not defined
-- | --
  | 1/14/2012:18:17.749 PM | except splunk.ResourceNotFound:
  | 1/14/2012:18:17.749 PM | File "/opt/splunk/etc/apps/website_monitoring/bin/modular_input.zip/modular_input/server_info.py", line 111, in is_on_shc
  | 1/14/2012:18:17.749 PM | return function(*args, **kwargs)
  | 1/14/2012:18:17.749 PM | File "/opt/splunk/etc/apps/website_monitoring/bin/modular_input.zip/modular_input/shortcuts.py", line 31, in wrapper
  | 1/14/2012:18:17.749 PM | if not cls.is_on_shc(session_key):
  | 1/14/2012:18:17.749 PM | File "/opt/splunk/etc/apps/website_monitoring/bin/modular_input.zip/modular_input/server_info.py", line 154, in is_shc_captain
  | 1/14/2012:18:17.749 PM | return function(*args, **kwargs)
  | 1/14/2012:18:17.749 PM | File "/opt/splunk/etc/apps/website_monitoring/bin/modular_input.zip/modular_input/shortcuts.py", line 31, in wrapper
  | 1/14/2012:18:17.749 PM | if ServerInfo.is_shc_captain(input_config.session_key) == False:
  | 1/14/2012:18:17.749 PM | File "/opt/splunk/etc/apps/website_monitoring/bin/modular_input.zip/modular_input/modular_input_base_class.py", line 961, in do_run
  | 1/14/2012:18:17.749 PM | self.do_run(in_stream, log_exception_and_continue=True)
  | 1/14/2012:18:17.749 PM | File "/opt/splunk/etc/apps/website_monitoring/bin/modular_input.zip/modular_input/modular_input_base_class.py", line 1092, in execute
  | 1/14/2012:18:17.749 PM | Traceback (most recent call last):
  | 1/14/2012:18:17.749 PM | 2020-01-14 17:18:17,749 ERROR Execution failed


```

**Comment by LukeMurphey** [comment]
Thanks for the heads up. This ought to be fixed in version 2.9.1 (https://splunkbase.splunk.com/app/1493/).
